templates/metadata/decoded/decoded.1.schema.json

```diff
@@ -3,7 +3,23 @@
   "id": "http://jsonschema.net",
   "type": "object",
   "properties": {
-    @METADATA_TELEMETRYINGESTION_1_JSON@,
+    @METADATA_INGESTIONTOPLEVEL_1_JSON@,
+    "metadata": {
```

Comment:
I refactor a bit in this PR such that the `metadata` struct always appears explicitly. This allows inserting the additional document_* attributes here, but also makes for the definitions of telemetry-ingestion and structured-ingestion to be more parallel. We factor out `uri` to its own include file instead, so that it can be used in the decoded schema and the telemetry-ingestion schema.